.github/workflows/build-deb.yml

```diff
@@ -0,0 +1,23 @@
+
+name: Build Deb Package
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_dispatch:
+  release:
+    types: ["published"]
+jobs:
+  build-deb-package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Build Debian package
+        uses: dawidd6/action-debian-package@v1
+        with:
+          # Optional, relative to workspace directory
+          source_directory: homeassistant-supervised
+          # # Optional, value from `debian/changelog` is used if not defined
+          # os_distribution: bullseye
+          # # Optional, target architecture, defaults to amd64
+          # cpu_architecture: all
```

Comment:
Do we need create an deb package per machine type?

Reply:
To get this action to work possibly but how I have everything configured now it should work on all